src/hotspot/share/utilities/chunkedList.hpp

```diff
@@ -28,7 +28,7 @@
 #include "memory/allocation.hpp"
 #include "utilities/debug.hpp"
 
-template <class T, MEMFLAGS F> class ChunkedList : public CHeapObj<F> {
+template <class T, MemTag F> class ChunkedList : public CHeapObj<F> {
```

Comment:
Parameter name should be updated.  Suggest `mem_tag`.

Reply:
How about MT here or just M?  I would make this a further change though.

Reply:
We can change this later in a followup.